executor/bin/check-version.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# USAGE: ./bin/check-version.sh [-q|--quiet]
+#
+# Returns 0 (true) if the currently-installed cook executor version
+# matches the source version, and 1 (false) otherwise.
+#
+# When using the quiet flag (-q|--quiet), no additional output is printed.
+#
+# Note that versions containing the case-insensitive string "SNAPSHOT" never match
+# (i.e., they're considered "unstable", and should always require a rebuild).
+
+case "$1" in
+    -q|--quiet)
+        shift
+        verbose=false
+        ;;
+    '')
+        verbose=true
+        ;;
+    *)
+        echo "USAGE: $0 [-q|--quiet]"
+        exit -1
+esac
+
+source_version=`python3 -c 'from cook._version import __version__; print(__version__)'`
```

Comment:
Can we ensure that this command works when the script is invoked from any directory?

Reply:
Done